### ISSUE TITLE
Example presentation with a footer

### ISSUE THREAD

**Issue by lornajane** [rst2pdf/rst2pdf.github.io#14]
We don't have any examples with footers but these are common for both documents and slide decks. Create an example showing how to add and style a footer.

**Comment by reynaldo22** [comment]
you want to add footer?

**Comment by lornajane** [comment]
Yes, I'd like an additional document in the examples collection that has a footer in it. There are some instructions in the `CONTRIBUTING` file that may help
